src/TextField/TextFieldLabel.js

```diff
@@ -4,15 +4,14 @@ import React, { PropTypes } from 'react';
 import { createStyleSheet } from 'jss-theme-reactor';
 import classNames from 'classnames';
 import { easing } from '../styles/transitions';
+import LabelBase from '../internal/LabelBase';
```

Comment:
I can take a shot at a generic label for `RadioGroup` for example one once I have this completed.  I'll probably PR separately.